examples/asr/emformer_rnnt/tedlium3/lightning.py

```diff
@@ -197,8 +198,8 @@ def training_step(self, batch: Batch, batch_idx):
     def validation_step(self, batch, batch_idx):
         return self._step(batch, batch_idx, "val")
 
-    def test_step(self, batch, batch_idx):
-        return self._step(batch, batch_idx, "test")
+    def test_step(self, batch_tuple, batch_idx):
+        return self._step(batch_tuple[0], batch_idx, "test")
```

Comment:
What is the significance of `[0]` here?
All the other changes appear cosmetic, but this one is different.

Reply:
The change aligns with [LibriSpeech recipe](https://github.com/pytorch/audio/blob/main/examples/asr/emformer_rnnt/librispeech/lightning.py#L177-L178)

The reason is that for test_dataloader, the text transcription is required for WER computation, hence the [_test_collate_fn](https://github.com/pytorch/audio/blob/main/examples/asr/emformer_rnnt/librispeech/lightning.py#L138) returns the transcript along with the Batch input.